src/AccessibilityInsights.Desktop/UIAutomation/A11yPatternFactory.cs

```diff
@@ -110,7 +111,10 @@ public static A11yPattern GetPatternInstance(A11yElement e, IUIAutomationElement
             }
             catch(Exception ex)
             {
-                ex.ReportException();
+                if (!(ex is TelemetryExcludedException))
```

Comment:
This filtering may be more appropriate in the IAxeTelemetry sink. @RobGallo, your thoughts?

Reply:
I agree. If it's going to be anywhere, it should be in the logger. It's still a little strange though that we're bothering to send an exception at all. But I suppose this might enable us to someday get telemetry for the exceptions we ignore as well, or possibly mark certain exceptions as known issues.